notebooks/data analysis/Pathfinder_system_other_columns.ipynb

```diff
@@ -0,0 +1,3769 @@
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "source.head()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'pandas.core.frame.DataFrame'>\n",
+      "Int64Index: 1206 entries, 0 to 364\n",
+      "Data columns (total 1 columns):\n",
+      " #   Column  Non-Null Count  Dtype \n",
+      "---  ------  --------------  ----- \n",
+      " 0   value   1206 non-null   object\n",
+      "dtypes: object(1)\n",
+      "memory usage: 18.8+ KB\n"
+     ]
+    }
+   ],
+   "source": [
+    "source.info()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "array(['Pathfinder Bestiary', 'Pathfinder Core Rulebook',\n       'Pathfinder Bestiary 2', 'Pathfinder Bestiary 3'], dtype=object)"
+     },
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "source.value.unique()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Skąd pochodzą potworki"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Details: podsumowanie\n",
+    "* **alignment**\n",
+    "    * do rozpakowania (abilities:mod case)\n",
+    "    * każdy potworek ma\n",
+    "    * czy są istotne?\n",
+    "    * może się przydać do if-ów\n",
+    "    * czy są istotne?\n",
+    "* blurb\n",
+    "    * puste\n",
+    "* **creatureType**\n",
+    "    * nie każdy potworek ma\n",
+    "    * może się przydać do if-ów\n",
+    "* **level**\n",
+    "    * do rozpakowania (abilities:mod case)\n",
+    "    * każdy potworek ma\n",
+    "    * CR\n",
+    "* privateNotes\n",
+    "    * puste\n",
+    "* **publicNotes**\n",
+    "    * każdy potworek ma\n",
+    "    * opis potworków\n",
+    "    * zbędne\n",
+    "* **source**\n",
+    "    * każdy potworek ma\n",
+    "    * książka z której jest potworek\n",
+    "    * raczej zbędne"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Resources"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\jolka\\OneDrive\\Dokumenty\\Inzynierka\\ML-for-TTRPG-game-design\\analysis_functions.py:38: FutureWarning: The default dtype for empty Series will be 'object' instead of 'float64' in a future version. Specify a dtype explicitly to silence this warning.\n",
```

Comment:
Dziwne, wcześniej się tak nie działo, nie wiem w czym ta wartość jest wyjątkowa

Reply:
no chyba trzeba gdzieś wyspecyfikować typ na object?

Możesz to zostawić, kiedyś się poprawi